### ISSUE TITLE
[Feature Request] Radials for Humanoid Animations/Poses

### ISSUE THREAD

**Issue by JustBuddy** [VRCFury/VRCFury#269]
I am trying to write a locomotion agnostic version of my Poses Extension system.
In order to avoid having thousands of buttons, and to enable in-betweens, poses extension-poses are bundled in animation clips which contain multiple poses each. These are then selected by a radial. As it seems, VRCFury can not handle a toggle playing Humanoid Animation clips with the modifier “Use Slider Wheel”.

Trying so will result in the whole animation clip just playing instead of adhering to the slider value, demonstrated in this video clip: 
https://github.com/VRCFury/VRCFury/assets/8122264/4140f0a6-01e7-4e41-9baa-541113b91ea1

Basically, the slider wheel should control the motion time here. A neat addition would also be to be able to influence the motion speed in some edge cases like dances. (ref. GoGo Loco Emotes)
